spec/rails_helper.rb

```diff
@@ -16,7 +16,7 @@
 Capybara.default_wait_time = 10
 
 require 'factory_girl'
-FactoryGirl.definition_file_paths = [File.expand_path("../factories", __FILE__)]
+FactoryGirl.definition_file_paths = [File.expand_path('../factories', __FILE__)]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.